hamocc/mo_sedmnt.F90

```diff
@@ -151,17 +151,18 @@ SUBROUTINE ini_sedmnt(kpie,kpje,kpke,omask,sed_por)
          ! 2d and 3d fields are not allocated in case of sedbypass
          ! so only initialize them if we are using the sediment
          CALL ini_sedmnt_por(kpie,kpje,kpke,omask,sed_por)
-      end if
+      endif
+	  
  END SUBROUTINE ini_sedmnt
 
-      !========================================================================
+ !========================================================================
  SUBROUTINE ini_sedmnt_por(kpie,kpje,kpke,omask,sed_por)
-      !
-      ! Initialization of:
-      !   - 3D porosity field (cell center and cell boundaries)
-      !   - solid volume fraction at cell center
-      !   - vertical molecular diffusion coefficients scaled with porosity
-      !
+!
```

Comment:
Admittedly, I am not really happy with loosing the indentation here for the comments, since they belong to what comes after, which is also indented. But that might be just my personal matter and if you prefer it like this, I'll adapt.

Reply:
@jmaerz - is your preference to have all comments not indented? Is that the general consensus for hamocc?

Reply:
No, just the opposite. I would want to have the comments aligned to the code - I was just about to suggest this as a general comment - but it is something to generally agree on.

Reply:
We totally agree. Best free-format practices recommend - 'Indent blocks by 2 characters. Where possible, comments should be indented with the code within a block'.

Reply:
I also generally agree - in this specific case, however, I was interpreting the comment as a header of the subroutine and therefore aligned it with the SUBROUTINE statement above

Reply:
Right, if it would have been written above the subroutine (which I wouldn't want) I would agree. Admittedly, here and also in other places, I would prefer indentation (there are a few more places, where this holds). But again, I am also fine, if you prefer it differently.